src/structures/APIMessage.js

```diff
@@ -176,6 +176,23 @@ class APIMessage {
       flags = this.options.flags != null ? new MessageFlags(this.options.flags).bitfield : this.target.flags.bitfield;
     }
 
+    let mentionUserIds;
+    let mentionRoleIds;
+    if (this.options.allowedMentions) {
+      if (this.options.allowedMentions.users) {
```

Comment:
seeing as `parse: ['users']` and `users: [...ids]` are mutually exclusive, should there be a check here to prevent both being present? or should this error be left to the API to throw back?

Reply:
API imo